src/codegen/tests/initialization_test/snapshots/rusty__codegen__tests__initialization_test__type_initializers__unresolvable_types_validation.snap

```diff
@@ -0,0 +1,11 @@
+---
+source: src/codegen/tests/initialization_test/type_initializers.rs
+expression: msg
+---
+error: Some initial values were not generated
+  ┌─ <internal>:1:1
```

Comment:
this one looks weird as well (like src/codegen/tests/initialization_test/snapshots/rusty__codegen__tests__initialization_test__type_initializers__struct_init_with_wrong_types_does_not_trigger_codegen_validation.snap)